src/mod_vcard_mnesia.erl

```diff
@@ -150,6 +151,23 @@ filter_fields([{SVar, [Val]} | Ds], Match, VHost)
 filter_fields([_ | Ds], Match, VHost) ->
     filter_fields(Ds, Match, VHost).
 
+%% Fulltext search is mnesia is something that is really-really wrong
+stringify_search_fields(#vcard_search{} = S) ->
+    S#vcard_search{
+      lfn = binary_to_list(S#vcard_search.lfn),
```

Comment:
Why is it necessary to convert everything like this? ☹️
How does it work that fulltext search would be better if it were lists instead?

Reply:
We moved binary_to_list from mod_vcard to mod_vcard_mnesia.

It's not possible to filter binaries by prefix using match specifications, so mod_vcard_mnesia would have to use lists.

It's not fulltext search. It's just matching prefix of a string.

Reply:
Ah, I see it now, cool 👍